### ISSUE TITLE
Demo still not working

### ISSUE THREAD

**Issue by ghost** [szcom/rnnlib#36]
Why is the demo not working?

**Comment by encrypted-soul** [comment]
Yes, it is still not working. The repository doesn't seem to be maintained anymore. The last commit is dated 2017.

**Comment by justinbass** [comment]
Did you get the data sets installed? https://github.com/justinbass/rnnlib/releases/tag/dataset
